### DISCUSSION TITLE
Missing Gear/Cog for Horizontal Filter Bar

### DISCUSSION THREAD

**Discussion by cjholzmann-1** [apache/superset#23457] [Q&A / Help]
Hello! Thank you to everyone who has contributed to this fantastic project.

I have read through as many discussions as I could find regarding the experimental horizontal filter bar feature but cannot seem to get it to work in either the latest master branch or **2.1.0rc3**. I have enabled the following feature flags but still cannot see the gear on the left-hand side:

'DASHBOARD_CROSS_FILTERS': True
'DASHBOARD_FILTERS_EXPERIMENTAL': True
'DASHBOARD_NATIVE_FILTERS': True
'HORIZONTAL_FILTER_BAR': True
'DASHBOARD_NATIVE_FILTERS_SET': True
    
Is there something else I am missing?

![image](https://user-images.githubusercontent.com/82157503/226983085-63bb272e-26bd-4ad1-af99-c8d817382a10.png)

**Comment by andrekef** [comment]
I set my `superset_config.py` to use 
```
FEATURE_FLAGS = {
    "HORIZONTAL_FILTER_BAR": True,
}
```
I am on Superset `Version: 3.1.0`

Hope this helps

**Comment by jooheeok** [comment]
If you change 'FEATURE_FLAGS' in the superset_config.py file, you must completely kill the server and run it again for it to take effect. This may be why the changed settings were not visible.

```
FEATURE_FLAGS = {
        "ENABLE_TEMPLATE_REMOVE_FILTERS" : True,
        "ENABLE_TEMPLATE_PROCESSING": True,
        "HORIZONTAL_FILTER_BAR": True,
        "DASHBOARD_NATIVE_FILTERS_SET": True,
        "DASHBOARD_NATIVE_FILTERS":True,
}
```

![image](https://github.com/user-attachments/assets/1547aea4-297a-441a-964a-d8ac3268de5c)
